README.md

```diff
@@ -96,12 +96,12 @@ Your `.env` file should look like this:
     TTN_DASHBOARD_MAIL_DOMAIN=example.com
     ### end of file ###
    ```
-5. `% docker-compose build`
```

Comment:
This message (lines 99/100) was added because users kept running into it; it's a common problem for people new to docker. I don't mind moving it to after step 6 (setting up apache), but it would be great to mention it. it's lamentably awkward to collaborate on a pull-request. However, I think I can make suggested changes via the online editor.

Reply:
I edited and added the message below.